_opensearch/query-dsl/query-filter-context.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: default
+title: Query and filter context
+parent: Query DSL
+nav_order: 5
+---
+
+# Query and filter context
+
+Queries consist of query clauses, which can be run in a [_filter context_](#filter-context) or [_query context_](#query-context). A query clause in a filter context asks the question "_Does_ the document match the query clause?" and returns matching documents. A query clause in a query context asks the question "_How well_ does the document match the query clause?" and returns matching documents, providing the relevance of each document in the [_relevance score_](#relevance-score).
+
+## Relevance score
+
+A _relevance score_ measures how well a document matches a query. It is a positive floating-point number that OpenSearch records in the `_score` metadata field for each document. A higher score corresponds to a more relevant document. While different query types calculate relevance scores differently, all query types take into account whether a query clause it is run in a filter or query context. 
```

Comment:
"...  all query types take into account whether a query clause *it is run in* a filter or query context."
meaning here?

... whether a query clause runs in a filter or query context. ?

Reply:
Do we have an example of the `_score` metadata field inside a document? Maybe a link somewhere else in the documentation?